src/components/comments/LikeDislikeComment.jsx

```diff
@@ -0,0 +1,176 @@
+import {Comment, Icon, Label} from "semantic-ui-react";
+import React, {Component} from "react";
+import axios from "axios";
+import PropTypes from 'prop-types';
+import {
+    get_axios_config,
+    URL
+} from "../../store/actions/async/CommentActions";
+import { infoToast } from '../UI/Toast/toast';
+
+
+class LikeDislikeComment extends Component {
```

Comment:
Is this the best place to do the `api` calls from @pbkabali?

Reply:
Was @Karuhanga's question answered?

Reply:
@Karuhanga I wanted the response of the API call to trigger modification of local component state for that particular comment. Remember the available object in the store is for a list of all comments on that article